### PR TITLE
Update pagination

### PR DESCRIPTION
**Pull request by manototh**

### CODE REVIEW

File: restapi/pagination.mdx [hunk 1]
```diff
@@ -21,12 +21,14 @@ Axiom uses cursor-based pagination for these endpoints. The parameters and mecha
 To use pagination with the Run Query endpoint:
 - Include the [`limit` operator](/apl/tabular-operators/limit-operator) in the APL query of your API request. The argument of this operator determines the number of events to display per page.
 - Use `sort by _time asc` or `sort by _time desc` in the APL query. This returns the results in ascending or descending chronological order. For more information, see [sort operator](/apl/tabular-operators/sort-operator).
+- Specify `startTime` and `endTime` in the body of your API request. If you don’t specify `endTime`, the results are different for each request which results in constantly changing pagination boundaries.
```
Comment (mhr3) [inline restapi/pagination.mdx]:
```suggestion
- Specify `startTime` and `endTime` in the body of your API request.
```

wouldn't go into details there

**Review by mhr3 — COMMENTED** [review]
The examples need to be updated as well with the two time fields

**Comment by manototh** [comment]
> The examples need to be updated as well with the two time fields

Yes, indeed, thanks. Please take a look if it works now. Also applied your suggestion.

File: restapi/pagination.mdx [hunk 2]
```diff
@@ -21,12 +21,14 @@ Axiom uses cursor-based pagination for these endpoints. The parameters and mecha
...
 - Use `sort by _time asc` or `sort by _time desc` in the APL query. This returns the results in ascending or descending chronological order. For more information, see [sort operator](/apl/tabular-operators/sort-operator).
+- Specify `startTime` and `endTime` in the body of your API request.
 
 ### Run Query (Legacy)
 
 To use pagination with the legacy Run Query endpoint:
 - Add the `limit` parameter to the body of your API request. The value of this parameter determines the number of events to display per page.
 - Add the `order` parameter to the body of your API request. In the value of this parameter, order the results by time in either ascending or descending chronological order. For example, `[{ "field": "_time", "desc": true }]`. For more information, see [order operator](/apl/tabular-operators/order-operator).
+- Specify `startTime` and `endTime` in the body of your API request. If you don’t specify `endTime`, the results are different for each request which results in constantly changing pagination boundaries.
```
Comment (manototh) [inline restapi/pagination.mdx]:
```suggestion
- Specify `startTime` and `endTime` in the body of your API request.
```

**Review by mhr3 — APPROVED** [review]
Thanks!

**Pull request merged**